css-two-column-layout/index.html

```diff
@@ -0,0 +1,106 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Challenge - Two Column Layout</title>
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+
+  <div class="container">
+
+    <div class="row">
+      <div class="column-full">
+        <h1>
+          SCALE, SPEED, AGILITY
+        </h1>
+        <p>
+          We've dedicated thousands of man hours to making healthcare institutions in the Helium
+
+          network run 10x more efficiently and deliver an unparalleled quality of care.
+        </p>
+      </div>
+    </div>
+
+    <div class="row">
+      <div class="column-one-half row">
+
+        <img src="images/software.svg" alt="software">
+        <div class="text">
+          <h3>
+            INTUITIVE,
+            SMART SOFTWARE
+          </h3>
+          <p>
+            We build prodcuts that transform how
+            healthcare facilities run across Africa
+            - making them radically more efficient
+            operrationally and financially, a complete
+            turnaround.
+          </p>
+        </div>
+      </div>
+
+      <div class="column-one-half row">
+        <img src="images/practice.svg" alt="practice">
+        <div class="text">
+          <h3>
+            PRACTICE MEDICINE, NOT
+            ADMINISTRATION.
+          </h3>
+          <p>
+            Our all-in-one electronic health records
+            (EHR), hospital managnement (HMIS) &amp;
+            billing system
+          </p>
+        </div>
+      </div>
+    </div>
+
+    <div class="row">
+      <div class="column-one-half row">
```

Comment:
A row shouldn't be a child of another row. It should go row > column > content. If you gave it a class of row, to copy some of it's styles, you should just give it to the `column-one-half` class, or create a new class.